apps/mobile/src/app/(home)/account/[account]/index.tsx

```diff
@@ -1,26 +1,26 @@
 import { useGetTokensList } from '@/components/widgets/tokens/use-get-tokens-list';
 import { useTotalBalance } from '@/hooks/balances/use-total-balance';
-import { MockedAccount } from '@/mocks/account.mocks';
 import { AccountLoader } from '@/store/accounts/accounts';
 import { useAccountByIndex } from '@/store/accounts/accounts.read';
+import { destructAccountIdentifier } from '@/store/utils';
 import { useLocalSearchParams } from 'expo-router';
 import { z } from 'zod';
 
 import { AccountLayout } from './account.layout';
 
 const configureAccountParamsSchema = z.object({
-  fingerprint: z.string(),
-  account: z.string().transform(value => Number(value)),
+  accountId: z.string(),
 });
 
 export default function AccountScreen() {
   const params = useLocalSearchParams();
 
-  const { fingerprint, account: accountIndex } = configureAccountParamsSchema.parse(params);
+  const { accountId } = configureAccountParamsSchema.parse(params);
+  const { fingerprint, accountIndex } = destructAccountIdentifier(accountId);
```

Comment:
@edgarkhanzadian - I changed my code to use the real `AccountStore` and started using `destructAccountIdentifier`. Checking back to your [original PR](https://github.com/leather-io/mono/pull/407/files) and I see that we increment `accountId` when we  use `destructAccountIdentifier` but I couldn't be sure that we increment it on creation.

Do we need to double check that it's working? I just want to make sure we aren't choosing the wrong account.